armi/reactor/components/complexShapes.py

```diff
@@ -140,6 +137,69 @@ def getCircleInnerDiameter(self, Tc=None, cold=False):
         return self.getDimension("holeOP", Tc, cold)
 
 
+class FilletedHexagon(basicShapes.Hexagon):
+    """Hexagon with rounded corners.
+
+    This shape has a hexagonal hole with rounded corners cut out of the center of it. By default,
+    that inner hole has a diameter of zero, making this a solid object with no hole.
+    """
```

Comment:
I agree with Alex, this description is not very clear to me. I have a suggestion, feel free to take it or leave it.
```suggestion
    """A hexagon with a hexagonal hole cut out of the center of it, where the corners of both the outer and inner hexagons are rounded.

    By default, the inner hole has a diameter of zero, making this a solid object with no hole.
    """
```